test/unit/liquid/drops/api_spec_drop_test.rb

```diff
@@ -26,6 +26,11 @@ def setup
     assert_nil api_spec.service
   end
 
+  test 'returns the host with port' do
+    @spec.expects(host_with_port: 'example.com:3000')
+    assert_equal 'example.com:3000', api_spec.host_with_port
+  end
```

Comment:
Can we add a test for cases where
1. the port is not part of the proxy endpoint
2. there's is no service linked to the apidoc
?

Reply:
I agree with Gui about this 👍

Reply:
>     1. the port is not part of the proxy endpoint

Meaning, the port is undefined or host_with_port = 'example.com'

>    2. there's is no service linked to the apidoc

How can I do that?

Reply:
> 1. there's is no service linked to the apidoc

`spec = FactoryBot.build(:api_docs_service, service: nil)`
@josemigallas I am not sure if this solves your doubt 😄